docs/decisions/0002-footer-links.rst

```diff
@@ -0,0 +1,168 @@
+container).
+
+.. _MFE's config mechanism: https://github.com/openedx/frontend-platform/blob/master/src/config.js
+
+Styles
+======
+
+Since the root of the footer component is rendered as a footer HTML element, it
+is unique enough to use ``footer`` as selector. To make it more convenient to
+style footer's children, we will add unique class names, which will simplify
+the SCSS selectors.
+
+These SCSS styles can be put in a custom `branding package`_ into
+``paragon/_overrides.scss``.
+
+.. _branding package: https://github.com/openedx/brand-openedx
+
+Consequences
+************
+
+.. This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
+Positive
+========
+
+* There will be an option to make the MFE footer look similar to the legacy UI
+  footer, without having to create and maintain a fork.
+* Can add custom text/urls for links, unlike what branding v1 API allows.
+
+Negative
+========
+
+* There are already parameters that allow configuring the footer links for the
+  legacy UI, which means that configuration will be duplicated (potentially in
+  multiple places), in cases when the aim is to make the links in the legacy UI
+  and MFE footers that same.
+
+Rejected Alternatives
+*********************
+
+.. This section lists alternate options considered, described briefly, with pros and cons.
+
+React component props API
+=========================
+
+Instead of reading the configuration values in the component, we could be
+reading it somewhere else, and passing the values as props to the react
+component. We couldn't identify any pros to this approach, and the cons are the
+same as the cons from decoupling.
+
+Styling using CSS variables
+===========================
+
+Exposing styling customizations through the CSS variables would be similar to
+styling via CSS classes, and it would be better aligned with the current
+philosophy of Paragon, since it prefers CSS variables over custom CSS classes.
+
+The cons however, is that you can only expose one CSS attribute per variable.
+This means that we would either heavily limit the styling options, or would
+have to create many variables for each element and CSS attribute.
+
+Brading API V1
+==============
+
+LMS already has an API endpoint which could be used to retrieve the links that
+are used in the footer (legacy UI footer uses the same API) in JSON format -
+``api/branding/v1/footer``.
+
+The pros of using this approach would be that legacy and MFE footer would use
+the same API and would have the same links, without having to duplicate the
+configuration.
+
+The cons of using this approach are:
+
+* It would be necessary to make an additional API call from an MFE to get the
+  links.
+* No custom links, only what is allowed by the branding API.
```

Comment:
It still requires an additional API call, but why not create a branding v2 API that does support custom links etc.

Reply:
@xitij2000 I tried looking up branding API v1 and v2 and couldn't find anything. My understanding is that the branding API v2 is [OEP-48](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0048-brand-customization.html), which only talks about the branding package. And I assumed v1 is the `api/branding/v1/...`. Could you please clarify for me, if my understanding is incorrect?

Reply:
What I meant is that instead of adding this to the MFE config API, we could create an v2 of the branding API as part of this work which would then be used by the new footer. If features are missing in v1, we can create v2.